README.md

```diff
@@ -45,18 +45,17 @@ Note: The section names are taken from the cyhy.conf
 Create compressed, encrypted, signed extract file with Federal CyHy data for integration with the Weathermap project.
 
 Usage:
-  COMMAND_NAME [--cyhy_section CYHY_SECTION] [--scan_section SCAN_SECTION] [--assessment_section ASSESSMENT_SECTION] [-v | --verbose] [-f | --federal] [-a | --aws] --config CONFIG_FILE [--date DATE]
+  COMMAND_NAME [--cyhy-config CYHY_CONFIG] [--scan-config SCAN_CONFIG] [--assessment-config ASSESSMENT_CONFIG] [-v | --verbose] [-a | --aws] --config CONFIG_FILE [--date DATE]
   COMMAND_NAME (-h | --help)
   COMMAND_NAME --version
 
 Options:
   -h --help                                                         Show this screen
   --version                                                         Show version
-  -x CYHY_SECTION --cyhy_section=CYHY_SECTION                       CyHy configuration section to use
-  -y SCAN_SECTION --scan_section=SCAN_SECTION                       Scan configuration section to use
-  -z ASSESSMENT_SECTION --assessment_section=ASSESSMENT_SECTION     Assessment configuration section to use
+  -x CYHY_CONFIG --cyhy-config=CYHY_CONFIG                       CyHy configuration section to use
```

Comment:
Please make the text on the right-hand side line up with the rest of the column.